src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/PaymentFailure.cshtml

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Should this at least containt some title for the sake of testers confirming the redirect was correct

Reply:
updated.